databases/backends/postgres.py

```diff
@@ -67,7 +71,6 @@ async def connect(self) -> None:
             host=self._database_url.hostname,
             port=self._database_url.port,
             user=self._database_url.username,
-            password=self._database_url.password,
```

Comment:
I'd rather do it with fewer changes. We can get away with the following:
1. Revert everything.
2.
```python
kwargs = dict(
    host=self._database_url.hostname,
    port=self._database_url.port,
    user=self._database_url.username,
    password=self._database_url.password,
    database=self._database_url.database,
)
kwargs.update(self._get_connection_kwargs())
self._pool = await asyncpg.create_pool(**kwargs)
```
You can then pass `password` to `options / self._options`.

Reply:
I agree. Like the updated code?